Subject: [PATCH] Remove weird file explorer
---
Index: LogicReinc.BlendFarm/Program.cs
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/LogicReinc.BlendFarm/Program.cs b/LogicReinc.BlendFarm/Program.cs
--- a/LogicReinc.BlendFarm/Program.cs	(revision 688d0dace0877cc7a25fe23050bab3a8fb3e1338)
+++ b/LogicReinc.BlendFarm/Program.cs	(date 1717169433886)
@@ -52,8 +52,6 @@
         {
             AppBuilder builder = AppBuilder.Configure<App>()
                 .UsePlatformDetect();
-            if (SystemInfo.IsOS(SystemInfo.OS_LINUX64))
-                builder = Avalonia.Dialogs.ManagedFileDialogExtensions.UseManagedSystemDialogs(builder);
             return builder.LogToDebug();
         }
     }
